### ISSUE TITLE
pcsx2 not working

### ISSUE THREAD

**Issue by shanewt0** [Nun-z/XboxRetailModeAppGuideLegacy#8]
Thanks for the detailed steps, that really helps!

I followed the steps using the build from source option and got everything to work as expected, except for pcsx2. I have the bios for pcsx2 set up and it works fine in dev mode. However, on retail mode, with the same bios, I tried loading the pcsx2 core and just run "start core", the app crashed immediately and exited out. Loading content didn't work either. I did this just now based on the latest commit on the retroarch repo (https://github.com/libretro/RetroArch/commit/49e99266f8c7fe363c6b33aa85a41b6516eb8a72). 

Can you try it out and see if pcsx2 works for you with the latest from the RetroArch repo? I saw there was a bounty couple months ago and looks like you already updated everything to work with ps2, maybe some newer updates on Retroarch-side caused issues with pcsx2?

**Comment by Nun-z** [comment]
Thank you for reporting this.  I'll have a chance to look at it tomorrow.

**Comment by shanewt0** [comment]
To add to this, I'm suspecting it's a file access issue. Using the retail version of the RetroArch I build with this guide, I can't write logs to locations outside of the RetroArch main folder. I tried setting logs to removable storage or the user download folder (which are the only places I can put the pcsx2 bios files) and no log file shows up. Then I tried setting the configuration file location to ones of these locations and saving the config file, and RetroArch complains that it cannot save the config file at those locations either. I guess that indicates that RetroArch cannot access files at those locations. Since I put the bios files in the user download folder, I guess that's why RetroArch cannot load the bios files there either. 

I tried using some other file explorer apps to attempt to drop those bios files into the RetroArch folder, but those file explorer apps cannot see inside the RetroArch installation folder either.
